dracut/30ignition/module-setup.sh

```diff
@@ -103,5 +103,7 @@ installkernel() {
 
      # required by applehv platform to read ignition file through vsock
      instmods -c vsock
+     instmods -c vmw_vsock_virtio_transport_common
+     instmods -c vmw_vsock_virtio_transport
```

Comment:
I'm guessing these may not be available on e.g. s390x; do we need to account for that?

Reply:
I checked that they were before approving this. :) (Still had that content from https://github.com/coreos/ignition/pull/1696#discussion_r1377732502.)